README.md

```diff
@@ -105,6 +105,26 @@ EOFF
 6. Stage the changes by running `osc addremove`.
 7. Commit the changes into OBS by running `osc ci`.
 
+## Submit packages
+
+To submit the latest version to `openSUSE Factory` do:
+
+```
+osc sr systemsmanagement:SCC:RMT rmt-server openSUSE:Factory
+```
+
+If a maintenance request is needed (e.g. `SLE 15`):
+
+```
+osc -A https://api.suse.de mr Devel:SCC:RMT rmt-server SUSE:SLE-15:GA
```

Comment:
Please add a note that this project is linked against systemsmanagement:SCC:RMT and never needs an osc commit to it ;-)

Reply:
done!